src/Dependencies/NativeDependencies.csproj

```diff
@@ -20,7 +20,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\build\Common.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>net50</TargetFramework>
```

Comment:
If this really needs to be changed to `net50`, then condition it to be `net462` on Windows and `net50` on non-Windows. Otherwise, the builds will have `warning NU1701: Package 'Microsoft.ApplicationInsights.Agent.Intercept 2.4.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project.` on Windows.

Reply:
@dividedmind will you be able to make this update?

Reply:
Done!

Reply:
Thanks!